Jint.Tests.PublicInterface/PublicInterfaceTests.cs

```diff
@@ -0,0 +1,100 @@
+using System.Collections.Concurrent;
+using Jint.Native;
+using Jint.Native.Function;
+using Xunit;
+
+namespace Jint.Tests.PublicInterface
```

Comment:
can use file-scoped namespace

Reply:
Removed the now-redundant using statement for Xunit.